index.src.html

```diff
@@ -161,16 +161,15 @@
         or <a>depth+video stream</a>.
       </p>
       <p>
-        Depth cameras usually produce 16-bit depth values per pixel. However,
-        neither the canvas drawing surface used to draw and manipulate 2D
-        graphics on the web platform nor the <code><a>ImageData</a></code>
-        interface used to represent image data support 16 bits per pixel. To
-        address the issue, this specification defines a conversion into a 8-bit
-        grayscale representation of a <a>depth map</a> for consumption by APIs
-        that are limited to 8 bits per pixel.
+        Depth cameras usually produce 16-bit depth values per pixel. So this
+        specification defines a 16-bit grayscale representation of a <a>depth
+        map</a>. There are two ways to access the depth values. First of all,
+        it's a <code>16bit</code> texture in [[WEBGL]] after uploading a
+        <a>depth map</a> to the texture. For instance, <code>RGB</code> and
+        <code>LUMINANCE_ALPHA</code> are <code>16bit</code> texture.
```

Comment:
move to "Implementation considerations"
let me defscribe how to pack format in another PR. It's needed in `v1` because `v1` has to support `WebGL1` as it piggyback `<video>`.